src/components/TabbedContainer/TabbedContainer.tsx

```diff
@@ -0,0 +1,87 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { color, typeface } from '../../styles';
+// import { H1 } from '../Headings';
```

Comment:
Just a quickie here - delete this comment 🚀

Reply:
Removed.